src/utils/src/lang.rs

```diff
@@ -0,0 +1,12 @@
+use fluent::{FluentBundle, FluentResource};
+
+pub fn translation_or_default(id: &str, resource: &FluentBundle<&'_ FluentResource>) -> String {
+    match resource.get_message(id) {
+        Some(msg) => {
+            let pattern = msg.value().expect("No Translation.");
+            let text = resource.format_pattern(&pattern, None, &mut vec![]);
+            text.into_owned()
+        }
+        None => String::from("Unknown"), // TODO: Get default from english version instead,
```

Comment:
or we can just replace with "missing translation" so it's obvious and gets fixed fast.

Reply:
English fallback would probably be best. we may add new features/text before every translation is done.
The english would provide more information than a "missing/unknown" message.

The government websites in Japan often work that way - show english if a translation exists, but fallback to japanese.